apps/chef_objects/src/chef_client.erl

```diff
@@ -127,26 +116,34 @@ set_updated(#chef_client{} = Object, ActorId) ->
     Now = chef_object_base:sql_date(now),
     Object#chef_client{updated_at = Now, last_updated_by = ActorId}.
 
+create_query(#chef_client{server_api_version = ?API_v0}) ->
```

Comment:
The chef_wm_base initialization sorts all that out, and if the client doesn't specify a version it sets ?API_v0.